psi4/driver/qcdb/vib.py

```diff
@@ -424,13 +423,14 @@ def vec_in_space(vec, space, tol=1.0e-4):
     space = ('T' if project_trans else '') + ('R' if project_rot else '')
     TRspace = _get_TR_space(mass, geom, space=space, tol=LINEAR_A_TOL)
     nrt = TRspace.shape[0]
-    text.append('  projection of translations ({}) and rotations ({}) removed {} degrees of freedom ({})'.format(
-        project_trans, project_rot, nrt, nrt_expected))
+    text.append(
+        f'  projection of translations ({project_trans}) and rotations ({project_rot}) removed {nrt} degrees of freedom ({nrt_expected})'
+    )
 
     P = np.identity(3 * nat)
     for irt in TRspace:
         P -= np.outer(irt, irt)
-    text.append(mat_symm_info(P, lbl='total projector') + ' ({})'.format(nrt))
+    text.append(mat_symm_info(P, lbl='total projector') + f' ({nrt})')
```

Comment:
I suppose if we're doing away with `lbl` in `harmonic_analysis`, it wouldn't hurt to `lbl -> label` here too. Perhaps a ToDo item?

Reply:
Sure. The other `lbl` was a field in the namedtuple QCAspect, where its counterpart in Datum was `label`, so it _had_ to change. This one's a fairly local argument, but I agree `label` is more readable.